Jenkinsfile

```
@@ -1 +1,5 @@
-buildPlugin(configurations: buildPlugin.recommendedConfigurations())
+buildPlugin(configurations: [
+    [ platform: "linux", jdk: "8"],
+    [ platform: "windows", jdk: "8"],
+    [ platform: "linux", jdk: "11", jenkins: "2.176.4", javaLevel: "8" ]
```

Looks like the plugin already depends on `2.176.4`, so you could leave that out here to pick it up as the default value:

```suggestion
    [ platform: "linux", jdk: "11", javaLevel: "8" ]
```